engine/execution/computation/manager.go

```diff
@@ -240,3 +226,24 @@ func (e *Manager) GetAccount(
 func (e *Manager) QueryExecutor() query.Executor {
 	return e.queryExecutor
 }
+
+func DefaultFVMOptions(chainID flow.ChainID, cadenceTracing bool, extensiveTracing bool) []fvm.Option {
+	options := []fvm.Option{
+		fvm.WithChain(chainID.Chain()),
+		fvm.WithReusableCadenceRuntimePool(
+			reusableRuntime.NewReusableCadenceRuntimePool(
+				ReusableCadenceRuntimePoolSize,
+				runtime.Config{
+					TracingEnabled: cadenceTracing,
+					// Attachments are enabled everywhere except for Mainnet
+					AttachmentsEnabled: chainID != flow.Mainnet,
```

Comment:
When I compare [the diff](https://github.com/onflow/flow-go/compare/master...bastian/update-stable-cadence-3) against master, I found we are missing two configs here:
```
					AccountLinkingEnabled: true,
					// Capability Controllers are enabled everywhere except for Mainnet
					CapabilityControllersEnabled: chainID != flow.Mainnet,
```
Do we need to add them?
<img width="1172" alt="image" src="https://github.com/onflow/flow-go/assets/811374/d789b8dd-e9b6-457e-827a-ac7c2e0b30f4">

Reply:
This is expected: 
- The `AccountLinkingEnabled` feature flag was removed from Cadence, as account links got replaced by account capability controllers
- The `CapabilityControllersEnabled` feature flag got removed, because the Capability Controllers API is the only capability management API